pkgs/development/libraries/qtutilities/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, qt5, cpp-utilities, cmake, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "qtutilities";
+  version = "5.13.0";
+
+  src = fetchFromGitHub {
+    owner = "Martchus";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "03drxwpr3xdh4hm8pkd5mhjs0mzhz6634ldyq78pml39ciqm51nl";
+  };
+
+  buildInputs = [ qt5.qtbase cpp-utilities ];
+  nativeBuildInputs = [ cmake qt5.qttools ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/Martchus/${pname}";
```

```suggestion
    homepage = "https://github.com/Martchus/qtutilities";
```
See also: https://github.com/NixOS/nixpkgs/pull/64453#pullrequestreview-259237998

---

Right, thanks!